Gems/AWSClientAuth/Code/Source/Authentication/GoogleAuthenticationProvider.cpp

```diff
@@ -83,7 +83,8 @@ namespace AWSClientAuth
 
         HttpRequestor::HttpRequestorRequestBus::Broadcast(&HttpRequestor::HttpRequestorRequests::AddRequestWithHeadersAndBody, m_settings->m_oAuthCodeURL
             , Aws::Http::HttpMethod::HTTP_POST, headers, body
-            , [this](const Aws::Utils::Json::JsonView& jsonView, Aws::Http::HttpResponseCode responseCode)
+            , [this](const Aws::Utils::Json::JsonView& jsonView, Aws::Http::HttpResponseCode responseCode,
+                [[maybe_unused]] AZStd::chrono::milliseconds roundTripTime)
```

Comment:
isn't `roundTripTime` an out parameter? ie shouldn''t this be declared as pass by reference?

Reply:
It a parameter in a callback. Could make it `const milliseconds&`, but it's really just a fancy int64.